test/visibility/import/duplicateMod.good

```diff
@@ -1 +1,2 @@
-duplicateMod.chpl:8: error: duplicate mention of the same module 'Sub'
+duplicateMod.chpl:7: In module 'Program':
+duplicateMod.chpl:8: error: cannot 'import' symbol 'Sub' as it is not defined in 'Sub'
```

Comment:
Since this caught my eye again, I'll mention it again:  I'd still de-quote 'import' here to reduce the amount of quoting in this message and since the English word seems great (and would still quote 'use' in the symmetric messages, where I'm imagining such cases could potentially be handled by some sort of ternary `(expr ? "import" : "'use'") that involves the quotes.